direct-hs-examples/src/ping.hs

```diff
@@ -39,14 +39,16 @@ handleCreateMessage client (D.Txt "select", _, room, _) = void $ D.sendMessage
 handleCreateMessage client (D.Txt "task", _, room, _) =
     void $ D.sendMessage client (D.TaskQ "高速化できる？" False) (D.talkId room)
 handleCreateMessage client (D.Txt "whoareyou", _, room, _) = do
-    Just me <- D.getMe client
+    me <- D.getMe client
     void $ D.sendMessage
         client
         (D.Txt $ "私は" `T.append` D.displayName me `T.append` "です")
         (D.talkId room)
 handleCreateMessage client (D.Txt "users", _, room, _) = do
     users <- D.getUsers client
-    let ans = T.unlines $ map D.displayName users
+    let
+        ans = T.unlines
+            $ map D.displayName (D.myself users : D.acquaintances users)
```

Comment:
Perhaps we should have a function like this:

```hs
getUsersList :: Client -> IO [User]
getUsersList client = do
  users <- getUsers client
  return (myself users : acquaintances users)
```

Reply:
I added helper function `usersList` to construct a user list easily. Does it make sense?

Reply:
According to oral discussion:

- Keep `getUsers`' signature `:: Client -> IO [User]` and behavior.
- Save `me` and `acquaintances` in separate `IORef`s.

Rationale: The `Users` type sounds weird. And actually we get `me` and `acquaintances` by separate RPC calls.